### ISSUE TITLE
Feature Request: Ignore()

### ISSUE THREAD

**Issue by jayd3e** [franela/goblin#60]
There should be a way to ignore tests if necessary.  A great use case for this is a situation where you need to account for a test in the future, but it doesn't apply now.  Your choices at the moment are to either delete it or comment the test out.  I would prefer to just be able to ignore it.  Another great use-case is integration tests with third parties that you don't always want to run.  You could implement it like so:

```
g.Ignore("Should add two numbers ", func() {
    g.Assert(1+1).Equal(2)
})
```

Testing frameworks in Scala have a similar feature.

**Comment by marcosnils** [comment]
Fixed by #62

**Comment by jayd3e** [comment]
@marcosnils awesome!  Excited to use this feature.  I have to say though that `Xit` is far less clear than a name like `Ignore`.  Not sure why it was selected.
